admin_interface/templates/admin/change_form.html

```diff
@@ -6,8 +6,9 @@
 
 {% get_admin_interface_setting "show_fieldsets_as_tabs" as show_fieldsets_as_tabs %}
 {% get_admin_interface_setting "show_inlines_as_tabs" as show_inlines_as_tabs %}
+{% admin_interface_use_changeform_tabs adminform inline_admin_formsets as admin_interface_use_changeform_tabs %}
```

Comment:
Since this template tag is used only once, maybe a with statement for setting the variable to check would be enough.